components/provisioning/warewulf/SPECS/warewulf.spec

```diff
@@ -49,6 +55,7 @@ Conflicts: warewulf-provision
 Conflicts: warewulf-ipmi
 
 %if 0%{?suse_version} || 0%{?sle_version}
+BuildRequires: distribution-release
```

Comment:
This breaks the build in our build system. This seems unnecessary anyway even if upstream has this line. Please remove it.